runbotics/runbotics-orchestrator-ui/src/views/process/ProcessBuildView/Modeler/ConfigureActionPanel/Actions/browser.actions.ts

```diff
@@ -730,11 +732,20 @@ const getBrowserActions: () => Record<string, IBpmnAction> = () => ({
             },
             uiSchema: {
                 'ui:order': ['input', 'output'],
+                input: {
+                    sessionDescription: {
+                        'ui:widget': 'TypographyWidget',
+                        'ui:options': {
+                            text: translate('Process.Details.Modeler.Actions.Browser.PrintToPdf.Session.Description'),
+                            variant: 'p',
```

Comment:
there is no variant like `p`
![image](https://user-images.githubusercontent.com/44121335/203570870-c6976f4a-dbeb-4c35-aefd-d936b6f9b3b3.png)